sass/layout/_webhooks.scss

```diff
@@ -353,3 +353,11 @@
         }
     }
 }
+
+.app__body {
+    .attachment {
+        .attachment__container {
+            border-left-color: var(--sidebar-header-bg);
```

Comment:
Thanks for the contribution, @ilhan-mstf!

I know the rule doesn't exactly match the old one, but would you mind moving this style to be up around line 107 in the `.post .attachment .attachment__container` rule? These files need a lot of cleanup eventually, and it would help a lot if this was near the other styling rules for that element. The `.app__body` part shouldn't matter here since message attachments always appear in a post anyway.

Reply:
Thank you,
I've updated as you requested. Can you please check again?